style.css

```diff
@@ -101,6 +125,225 @@ body {
   margin-right: 5px;
 }
 
-.flex-row {
+.work-section {
+  width: 375px;
+  height: 2788px;
+  display: grid;
+  text-align: left;
+  padding: 25px;
+  gap: 88px;
+  margin-bottom: 90px;
+  background-color: #fff;
+}
+
+.work-section ul {
+  display: flex;
+  justify-content: flex-start;
+  padding-left: 5px;
+}
+
+.work-section ul li {
+  list-style: none;
+  margin-right: 15px;
+  font-size: 13px;
+  padding-bottom: 20px;
+}
+
+.work1-title {
+  text-align: left;
+  font-size: 20px;
+}
+
+.work1-title > h2 {
+  margin-top: 5px;
+  margin-bottom: 5px;
+  padding-right: 21px;
+  line-height: 40px;
+  color: #172b4d;
+}
+
+.arrow {
+  margin-left: 138px;
+}
+
+.work1-detail ul li a {
+  color: #2e4662;
+}
+
+.text-decoration {
+  text-decoration: none;
+}
+
+.font-color {
+  color: #344563;
+}
+
+.work1-description {
+  text-align: left;
+  color: #344563;
+  padding-right: 39px;
+}
+
+.work-description-detail ul li {
+  font-size: 12px;
+  background-color: #eaebfe;
+  padding: 4px 12px;
+  border-radius: 8px;
+}
+
+.blue-color {
+  color: #396df2;
+}
+
+.btn-see-project {
   display: flex;
+  justify-content: flex-start;
+  margin-top: 25px;
+}
+
+.btn:enabled {
+  font-size: 19px;
+  text-align: start;
+  padding: 8px 1px;
```

Comment:
- [ ] Kindly change the `padding` to match that of the Figma template. Currently what you have doesn't make a great user experience. See the screenshot below for more details:

Figma:
<img width="335" alt="Screenshot 2023-04-02 at 03 03 17" src="https://user-images.githubusercontent.com/107192076/229328897-71376650-45af-4cd7-bfd4-9dd5d2d3840d.png">

Your design:
<img width="326" alt="Screenshot 2023-04-02 at 03 02 21" src="https://user-images.githubusercontent.com/107192076/229328905-efe9cb25-4a87-4cce-bffb-22c9330cb695.png">

- [ ] Also, please align the button text to the center for a great user experience.